README.md

```diff
@@ -199,19 +178,51 @@ WARNING:root:APPROX_COUNT_DISTINCT does not support accuracy
 SELECT APPROX_COUNT_DISTINCT(a) FROM foo
 ```
 
-### Rewrite Sql
-Modify sql expressions like adding a CTAS
+### Build and Modify Sql
+SQLGlot supports incrementally building sql expressions.
+
+```python
+from sqlglot import select
+
+select("*").from_("y").where("x > 1").sql()
+```
+
+```sql
+SELECT * FROM y WHERE x > 1
+```
+
+You can also modify a parsed tree:
+
+```python
+from sqlglot import parse_one
+
+parse_one("SELECT x FROM y").from_("z").sql()
```

Comment:
should from_ add a from? so it should be y, z

Reply:
Currently, `FROM` always overrides. I can change it to append like the others.